lib/internal/bootstrap/pre_execution.js

```diff
@@ -147,12 +147,10 @@ function setupWarningHandler() {
 
 // https://fetch.spec.whatwg.org/
 function setupFetch() {
-  if (!getOptionValue('--experimental-fetch')) {
+  if (getOptionValue('--no-experimental-fetch')) {
     return;
   }
 
-  emitExperimentalWarning('Fetch');
```

Comment:
Maybe we will want to keep the warning, but I didn't find an easy way to do it, so leaving that for later:
- Warning on first call would be doable for `fetch` by wrapping it but not really for the 3 other classes (maybe fetch is enough)
- Warning on first access would be a problem for feature detection.

Reply:
I strongly feel that at this implementation state we should keep a runtime warning - at least until the implementation is further along.

I think for `fetch` it's probably enough to warn on first use.

Reply:
@benjamingr what do you think about the other APIs (`Headers` for example)? I'm not sure how to implement a runtime warning for them.

Reply:
I think it's a lot less important for them since they'll likely be used with `fetch` anyway

Reply:
Warning added.